ansible/roles/mailhog/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Restart MailHog
```

Comment:
The handler ensures that future changes to the config file will restart the service.